diff --git a/node_modules/@astrojs/starlight/integrations/shared/pathToLocale.ts b/node_modules/@astrojs/starlight/integrations/shared/pathToLocale.ts
index aa8748c..9ceb94d 100644
--- a/node_modules/@astrojs/starlight/integrations/shared/pathToLocale.ts
+++ b/node_modules/@astrojs/starlight/integrations/shared/pathToLocale.ts
@@ -17,7 +17,7 @@ export function pathToLocale(
 		astroConfig,
 	}: {
 		starlightConfig: { locales: StarlightConfig['locales'] };
-		astroConfig: { root: URL; srcDir: URL };
+		astroConfig: { root: URL | string; srcDir: URL };
 	}
 ): string | undefined {
 	const srcDir = new URL(astroConfig.srcDir, astroConfig.root);
diff --git a/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts b/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts
index 5af454f..34bb7bb 100644
--- a/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts
+++ b/node_modules/@astrojs/starlight/user-components/rehype-file-tree.ts
@@ -168,7 +168,7 @@ function getFileIcon(fileName: string) {
 
 /** Return the icon name for a file based on its file name. */
 function getFileIconName(fileName: string) {
-	let icon = definitions.files[fileName];
+	let icon = definitions.files[fileName] as string | undefined;
 	if (icon) return icon;
 	icon = getFileIconTypeFromExtension(fileName);
 	if (icon) return icon;
