diff --git a/node_modules/starlight-links-validator/libs/remark.ts b/node_modules/starlight-links-validator/libs/remark.ts
index b3239f3..a93aa26 100644
--- a/node_modules/starlight-links-validator/libs/remark.ts
+++ b/node_modules/starlight-links-validator/libs/remark.ts
@@ -56,7 +56,12 @@ export const remarkStarlightLinksValidator: Plugin<[{ base: string; srcDir: URL
             break
           }
 
-          fileHeadings.push(slugger.slug(content))
+          // Remove trailing -, given that a heading can contain trailing markup e.g. a deprecation <Badge>
+          // GitHub Slugger considers it as content and adds a -
+          // Whereas Astro Markdown renderer removes this trailing - from the DOM node's id
+          // See: tests/markup-in-heading
+          const headingSlug = slugger.slug(content).replace(/-$/, '')
+          fileHeadings.push(headingSlug)
 
           break
         }
